code/modules/mob/living/carbon/carbon.dm

```diff
@@ -625,7 +625,7 @@
 
 /mob/living/carbon/movement_tally_multiplier()
 	. = ..()
-	if(!istype(loc, /turf/space) && !reagents.has_any_reagents(list(HYPERZINE,COCAINE)))
+	if(!istype(loc, /turf/space))
 		for(var/obj/item/I in get_clothing_items())
 			if(I.slowdown <= 0)
 				testing("[I] HAD A SLOWDOWN OF <=0 OH DEAR")
```

Comment:
Potential debug leftover here?

Reply:
does that show to everyone or only admins?

Reply:
That only shows up in the logs